docs/guide/README.md

```diff
@@ -0,0 +1,12 @@
+Assets
+======
+
+An asset in Yii is a file that can be referenced on a web page. It can be a CSS file, a JavaScript file, an image or video file, etc. The assets are configured in diretories non-public, which can be accessed once it is published.
```

Comment:
```suggestion
An asset in Yii is a file that can be referenced on a Web page. It can be a CSS file, a JavaScript file, an image or video file, etc. These files are usually placed in non-public directory that is then becomes accessible from public directory either by copying or by symlinking. This preparation step is called publishing.
```